source/updateCheck.py

```diff
@@ -119,7 +119,8 @@ def checkForUpdate(auto: bool = False) -> Optional[Dict]:
 		"version": versionInfo.version,
 		"versionType": versionInfo.updateVersionType,
 		"osVersion": winVersionText,
-		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64",
+		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") in ("AMD64", "ARM64"),
+		"osArchitecture": os.environ.get("PROCESSOR_ARCHITEW6432"),

```

Comment:
this change will require server side changes

Reply:
This should be safe to merge as-is, with changes to the server implemented later

Reply:
It's worth raising that this is additional user tracking.

I don't think it is a big leap from tracking x64 vs x86.
The only difference is tracking which x64-bit architecture is in use, out of (AMD64, ARM64, IA64).
One potential negative - considering ARM64 and IA64 are more unusual, getting a thumbprint of a user based on their update check might become easier in theory.